openasip/src/applibs/LLVMBackend/TCETargetMachine.cc

```diff
@@ -294,7 +299,7 @@ TCEPassConfig::addPreISel() {
     if (OptLevel != CodeGenOpt::None) {
         // get some pass lists from llvm/Support/StandardPasses.h from 
         // createStandardLTOPasses function. (do not add memcpyopt or dce!)
-        addPass(createInternalizePass());
+        //addPass(createInternalizePass());
```

Comment:
If it's not needed, just remove it compeltely.

Reply:
Removed.